src/style.css

```diff
@@ -0,0 +1,95 @@
+@font-face {
+  font-family: "Avenir-Book";
+  src: url("fonts/Avenir-Book.ttf");
+}
+
+@font-face {
+  font-family: "Avenir-Heavy";
+  src: url("fonts/Avenir-Heavy.ttf");
+}
```

Comment:
there is no need to give distinct names to these font-faces, name should be the same so that it can be used the following way:
```
.class1 { // uses Avenir-Book
  font-family: Avenir, ...;
  font-weight: 300;
}

.class2 { // uses Avenir-Heavy
  font-family: Avenir, ...;
  font-weight: 900;
}
```

Reply:
also, you'd rather put `font-style: normal` here to avoid putting it several times down below